.gitignore

```diff
@@ -348,3 +349,11 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+<<<<<<< Updated upstream
```

Comment:
I don't think we want these git change markers checked in... guessing this was not on purpose?